src/output-json-flow.c

```diff
@@ -213,6 +213,44 @@ void EveAddFlow(Flow *f, JsonBuilder *js)
     jb_set_string(js, "start", timebuf1);
 }
 
+
+#if defined(ENABLE_ETM)
+void EveAddEncryptedTrafficMetaData(const Flow *f, JsonBuilder *js)
+{
+    if ((f->splt.first_epoch_msec > 0) && (f->splt.splt_count > 0)) {
+        uint32_t i = 0;
+        FlowEncryptedTrafficFinalize(f);
```

Comment:
what is the purpose of finalize here? Why is it updating the flow from the output path? What happens if we have multiple eve instances and this gets called multiple times for the same packet?